ci-scripts/package.sh

```diff
@@ -31,34 +27,38 @@ if [[ "$OSTYPE" == "darwin"* ]]; then
   TARGET=$ARCH"-apple-darwin"
 fi
 
-echo "Evaluating program arguments "$@
+echo "Evaluating program arguments '$@'"
```

Comment:
Wouldn't the single quotes make `$@` a string literal and not expand the list of args?

Reply:
Not when it's already in a double quote for echo. Or at least, works on my machine :D